game.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+# player class
+class Player
+  attr_accessor :name, :hand, :bank, :total, :bet
+
+  def initialize(name, bank_roll)
+    @name = name
+    @hand = []
+    @bank_roll = bank_roll
+    @total = 0
+    @bet = 0
+  end
+
+  def add_card_in_hand(card)
+    card.score = 1 if card.score == 11 && @total > 11
+    @total += card.score
+    @hand << card
+  end
+
+  def intrdouce
+    puts "I am #{@name}, I have $#{@bank_roll} left."
+  end
+end
+
+# card class
+class Card
+  attr_accessor :value, :suit, :score
+
+  def initialize(value, suit, score)
+    @value = value
+    @suit = suit
+    @score = score
+  end
+
+  def show_card
+    puts "#{@value}-#{@suit} (#{@score})"
+  end
+end
+
+# Deck class
+class Deck
+  def initialize
+    @values = ['A', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K']
+    @suits = ['clubs', 'spades', 'hearts', 'diamonds']
+    @cards = []
+    build_deck
+  end
+
+  def build_deck
+    @values.each do |value|
+      if value.is_a? Integer
+        score = value
+      elsif value == 'A'
+        score = 11
+      else # J, Q, K
+        score = 10
+      end
```

Comment:
This logic will be moved to Card class if we plan to incorporate [this suggestion](https://github.com/MuddassirAhmed/black-jack-ruby/pull/1/files#r948462855)

Reply:
move to `Card` class